internal/gatewayapi/route.go

```diff
@@ -50,6 +51,31 @@ func (t *Translator) ProcessHTTPRoutes(httpRoutes []*v1beta1.HTTPRoute, gateways
 	return relevantHTTPRoutes
 }
 
+func (t *Translator) ProcessGRPCRoutes(grpcRoutes []*v1alpha2.GRPCRoute, gateways []*GatewayContext, resources *Resources, xdsIR XdsIRMap) []*GRPCRouteContext {
+	var relevantGRPCRoutes []*GRPCRouteContext
+
+	for _, g := range grpcRoutes {
+		if g == nil {
+			panic("received nil httproute")
```

Comment:
s/httproute/grpcroute/

Reply:
fixed in https://github.com/envoyproxy/gateway/pull/962